assets/app/views/createRoute.html

```diff
@@ -0,0 +1,52 @@
+<div class="content">
+  <div class="container">
+    <div class="col-md-12">
+      <ol class="breadcrumb">
+        <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
+        <li class="active"><strong>Add Route</strong></li>
+      </ol>
+      <alerts alerts="alerts"></alerts>
+      <h1>Add Route</h1>
+      <div ng-if="serviceName">
+        <span class="help-block">Create a route for service {{serviceName}}.</span>
+      </div>
+      <div ng-if="!serviceName">
+        <span class="help-block">Select a service and enter route details.</span>
```

Comment:
I checked what the help for oc expose says: "Expose containers internally
as services or externally via routes"   so it does say external there.

On Tue, Nov 24, 2015 at 5:04 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/createRoute.html
> <https://github.com/openshift/origin/pull/5997#discussion_r45801609>:
>
> > @@ -0,0 +1,52 @@
> > +<div class="content">
> > +  <div class="container">
> > +    <div class="col-md-12">
> > +      <ol class="breadcrumb">
> > +        <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
> > +        <li class="active"><strong>Add Route</strong></li>
> > +      </ol>
> > +      <alerts alerts="alerts"></alerts>
> > +      <h1>Add Route</h1>
> > +      <div ng-if="serviceName">
> > +        <span class="help-block">Create a route for service {{serviceName}}.</span>
> > +      </div>
> > +      <div ng-if="!serviceName">
> > +        <span class="help-block">Select a service and enter route details.</span>
>
> I get what you're saying, but at some point we have to differentiate
> between internal and external, right? You probably don't want to make your
> DB external. Maybe it's good if it's slightly scary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5997/files#r45801609>.
>